From 93e9aedaa75a3fff50f36c85147f53f75c3f289f Mon Sep 17 00:00:00 2001
From: Lucas Adriano Salles <83602841+luc-salles@users.noreply.github.com>
Date: Fri, 11 Oct 2024 12:58:14 -0400
Subject: [PATCH] hwdb: fix key toggle touchpad and programmable buttom for
 Positivo V142N (#34725)

(cherry picked from commit c2f3cb2676f70823afe7f0fcff8720023dedfab4)

Resolves: RHEL-49810
---
 hwdb.d/60-keyboard.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index f1af5771f5..b8bd7f0fb5 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -2231,6 +2231,8 @@ evdev:name:AT Translated Set 2 keyboard:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn
 evdev:name:SIPODEV USB Composite Device:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn*:pvr*:rvn*:rnK142*
 # Positivo Motion (N14ZP6, N14ZP7)
 evdev:name:SIPODEV USB Composite Device:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn*:pvr*:rvn*:rnN14ZP[67]*
+# Positivo Motion (V142N)
+evdev:name:SINO WEALTH USB KEYBOARD:dmi:bvn*:bvr*:svnPositivoInformaticaSA:pn*:pvr*:rvn*:rnV142N*
 # Positivo Motion (N14EP6)
 evdev:name:SIPODEV USB Composite Device:dmi:bvn*:bvr*:br*:svnPositivoTecnologiaSA:pn*:pvr*:rvn*:rnN14EP6*
 # Positivo Motion (CW14Q01P) (CW14Q01P-V2)
